challenge_2/python/sarcodian/src/challenge_2.py

```diff
@@ -0,0 +1,12 @@
+def pick_char(array0):
+    unique = []
+    duplicate = []
+    for i in array0:
+        if i in duplicate:
+            pass
+        elif i in unique:
+            duplicate.append(i)
+            unique.remove(i)
+        else:
+            unique.append(i)
+    return unique[0]
```

Comment:
Using a set or dict for duplicates would be better because you expect most elements to go in there and scanning through a list to check if an element is in there and to remove an element is more expensive than doing those with a dict or set.